src/notation/view/notationswitchlistmodel.h

```diff
@@ -49,6 +49,11 @@ class NotationSwitchListModel : public QAbstractListModel, public async::Asyncab
     Q_INVOKABLE void load();
     Q_INVOKABLE void setCurrentNotation(int index);
     Q_INVOKABLE void closeNotation(int index);
+    Q_INVOKABLE void closeOtherNotations(int index);
```

Comment:
What does this function do?

Reply:
It closes all notations except the specified one and the master notation. (The name of this method is based of the name of this functionality in the UI.)